core/frontend/src/components/notifications/ConfigMenu.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <v-card
+    elevation="1"
+    width="250"
+  >
+    <v-list>
+      <v-list-item>
+        <v-switch
+          :value="showOld"
+          color="blue lighten-1"
+          @change="emitShowOldChange"
+        />
+        <div>Show old messages</div>
+      </v-list-item>
+    </v-list>
+  </v-card>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+
+export default Vue.extend({
+  name: 'NotificationsConfigMenu',
+  props: {
+    showOld: {
+      type: Boolean,
```

Comment:
How eslint is not detecting this ?? boolean should be not capitalized.

Reply:
It's not that. Read [this](https://vuejs.org/v2/guide/components-props.html#Prop-Types).